mavros/launch/px4_config.yaml

```diff
@@ -214,18 +214,12 @@ mocap:
 # odom
 odometry:
   in:
-    frame_id: "odom"
-    child_frame_id: "base_link"
-    frame_tf:
-      local_frame: "local_origin_ned"
-      body_frame_orientation: "flu"
+    fcu_odom_parent_id_des: "local_origin"    # frame in which pose is expressed 
+    fcu_odom_child_id_des: "fcu"              # frame in which twist is expressed
   out:
-    frame_tf:
```

Comment:
Keep the `tf` namespace.

Reply:
These frames are not used to produce tf transforms. They are just parent and child frames of the odom messages.